packages/xdl/src/start/ExpoUpdatesManifestHandler.ts

```diff
@@ -131,6 +218,7 @@ export function getManifestHandler(projectRoot: string) {
         projectRoot,
         host: req.headers.host,
         platform: getPlatformFromRequest(req),
+        acceptSignature: req.headers['expo-accept-signature'] !== undefined,
```

Comment:
I'm not sure `expo-accept-signature:` (empty string) should count as enabling signatures. This looks OK for now but we probably will want to revisit it.

We didn't add much about signing to the Expo Updates spec and so I don't think the definition of a valid boolean value was ever written down in our spec. Interestingly, I can't seem to find common HTTP headers that have boolean values. Perhaps this should be a version string...

Reply:
For classic update served in expo-cli, it just does a simple boolean check against the value of the header: https://github.com/expo/expo-cli/blob/master/packages/xdl/src/start/ManifestHandler.ts#L364

The type says it can be `string | string[] | undefined`. I can change this to a simple boolean check.